frontend/src/components/ModelBox/ModelConfigurationSlider.tsx

```diff
@@ -12,6 +12,9 @@ function ModelConfigurationSlider({
 	config: CustomChatModelConfiguration;
 }) {
 	const [value, setValue] = useState<number>(config.value);
+	const [currentConfiguredValue, setCurrentConfiguredValue] = useState<number>(
+		config.value
+	);
```

Comment:
Fairly certain we don't need this state, because config.value is in props. To know if the slider value has changed on commit, just compare it to config.value.